client/src/styles.scss

```diff
@@ -1,5 +1,10 @@
 /* You can add global styles to this file, and also import other style files */
 
+html, body {
+	height: 100%;
+	position: relative;
```

Comment:
I do. Take a look at the picture I'm attaching, this is how it looks when the `height: 100%;` is removed from the `html` element.
The `body` element looks at it's parent(`html`) to see what 100% means.
[See this discussion.](http://stackoverflow.com/questions/6654958)
![screen shot 2016-11-23 at 19 35 44](https://cloud.githubusercontent.com/assets/11275466/20573421/57b6d66e-b1b8-11e6-897c-a5fe1e51e9aa.png)